Library/Formula/yank.rb

```diff
@@ -0,0 +1,18 @@
+class Yank < Formula
+  desc "Yank terminal output to clipboard"
+  homepage "https://github.com/mptre/yank"
+  url "https://github.com/mptre/yank/archive/v0.4.0.tar.gz"
+  # version "0.4.0"
+  sha256 "1adf1b07f7cb9401daeed7a05bad492db8ed77ead4728d9b45f541d56bc2e8c5"
+
+  def install
+    ENV["PREFIX"] = prefix
+    ENV["YANKCMD"] = "pbcopy"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/yank", "-v"
+    system "man", "yank"
```

Comment:
We were able to test similar formulae using `expect`, like [`scrypt.rb`](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/scrypt.rb#L21-L38). See this to understand how it works and how to use `autoexpect` to generate these scripts: http://www.admin-magazine.com/Articles/Automating-with-Expect-Scripts

If you don’t find any way to test this we’ll accept the `yank -v` test.